src/app/cube/home/learning-object-info/hierarchies/hierarchies.component.scss

```diff
@@ -34,6 +34,7 @@ h1 {
   //Wrapper Outer Parent
   display: inline-block;
   margin-top: 1%;
+  max-width: 300px;
```

Comment:
Just curious, why was this added?

Reply:
I'm not sure how those got there, I can remove them.

Reply:
Just the max-width I think should have been deleted, not the entire file.

Reply:
yeah the entire should not have been deleted.

Reply:
Oh no, sorry guys :O

Reply:
All good! Mistakes happen. This is why we have this review process :)